llvm/test/Transforms/LoopIdiom/AArch64/byte-compare-index.ll

```diff
@@ -2,6 +2,9 @@
 ; RUN: opt -aarch64-lit -aarch64-lit-verify -verify-dom-info -mtriple aarch64-unknown-linux-gnu -mattr=+sve -S < %s | FileCheck %s
 ; RUN: opt -aarch64-lit -simplifycfg -mtriple aarch64-unknown-linux-gnu -mattr=+sve -S < %s | FileCheck %s --check-prefix=LOOP-DEL
 ; RUN: opt -aarch64-lit -mtriple aarch64-unknown-linux-gnu -S < %s | FileCheck %s --check-prefix=NO-TRANSFORM
+; RUN: opt -p aarch64-lit -aarch64-lit-verify -verify-dom-info -mtriple aarch64-unknown-linux-gnu -mattr=+sve -S < %s | FileCheck %s --check-prefix=CHECK-NPM
```

Comment:
do you know why the new PM pass gives different results? ideally we wouldn't be adding thousands of CHECK lines

Reply:
makes sense, but can figure out how to not add thousands of CHECK lines just to make sure the new PM pass works? either create a new test, or change the test to already be in loop simplify and LCSSA form (committed in a separate change and rebase this change on that)